app/src/test/java/com/babylon/wallet/android/domain/model/TokensPriceSorterTest.kt

```diff
@@ -1,92 +0,0 @@
-package com.babylon.wallet.android.domain.model
-
-import com.babylon.wallet.android.domain.model.assets.Asset
-import com.babylon.wallet.android.domain.model.assets.AssetPrice
-import com.babylon.wallet.android.domain.model.assets.FiatPrice
-import com.babylon.wallet.android.domain.model.assets.SupportedCurrency
-import com.babylon.wallet.android.domain.model.assets.Token
-import com.babylon.wallet.android.domain.model.assets.TokensPriceSorter
-import com.babylon.wallet.android.domain.model.resources.Resource
-import com.babylon.wallet.android.domain.model.resources.XrdResource
-import com.babylon.wallet.android.domain.model.resources.metadata.Metadata
-import com.babylon.wallet.android.domain.model.resources.metadata.MetadataType
-import junit.framework.TestCase.assertEquals
-import org.junit.Test
-import rdx.works.profile.derivation.model.NetworkId
-
-class TokensPriceSorterTest {
```

Comment:
why did you remove this test? 😮

Reply:
Thanks not sure what happened here. I am putting it back